sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -353,7 +370,88 @@ struct graphs_support;
 } // namespace device
 } // namespace info
 
-class node {};
+enum class dynamic_parameter_type {
```

Comment:
this proposal should be compatible with https://github.com/intel/llvm/pull/12098